lints/lint_dnsname_bad_character_in_label.go

```diff
@@ -33,13 +33,15 @@ func (l *DNSNameProperCharacters) labelContainsBadCharacters(domain string) bool
 
 func (l *DNSNameProperCharacters) Execute(c *x509.Certificate) *LintResult {
 	if c.Subject.CommonName != "" {
```

Comment:
It doesn't look like this correctly handles `*.?.bad.com`

Reply:
AH, good catch.

Reply:
i fixed that btw.